Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/client/clientcmd/loader.go

```diff
@@ -35,8 +35,13 @@ const (
 	RecommendedConfigPathEnvVar = "KUBECONFIG"
 )
 
-// ClientConfigLoadingRules is a struct that calls our specific locations that are used for merging together a Config
+// ClientConfigLoadingRules is a collection that allows multiple sets of loading rules
 type ClientConfigLoadingRules struct {
+	Rules []ClientConfigLoadingRule
```

Comment:
Right, so that we can add the OpenShift-specific config rules to the ones already existing for Kube to build a single client. Also, notice we are proposing to remove the --kubeconfig flags and just support --config. 

Set 1: "--config" "OPENSHIFTCONFIG" "./.openshiftconfig" "~/.config/openshift/.config"
Set 2: ""         "KUBECONFIG"      "./.kubeconfig"      "~/.kube/.kubeconfig"

They end up making one single queue of priorities though, so maybe I could refactor to simplify and reflect that.

Set:
--config
OPENSHIFTCONFIG
./.openshiftconfig
~/.config/openshift/.config
KUBECONFIG
./.kubeconfig
~/.kube/.kubeconfig


----- Original Message -----
> >  type ClientConfigLoadingRules struct {
> > +	Rules []ClientConfigLoadingRule
> 
> Is this so we can have --kubeconfig and --osconfig to build two distinct
> clients, or so we can recognize .openshiftconfig in addition to .kubeconfig
> to build a single client?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/992/files#r25071314